app/controllers/moments_controller.rb

```diff
@@ -8,6 +9,19 @@ class MomentsController < ApplicationController
   # GET /moments
   # GET /moments.json
   def index
+    if current_user
+      @user_logged_in = true
+
+      period = 'day'
+      # +1 day buffer to ensure we include today as well
+      end_date = Date.current + 1.days
+      start_date = get_start_by_period(period, end_date)
+
+      @react_moments = Moment.where('userid = ?', current_user.id).group_by_period(period, :created_at, range: start_date..end_date).count
```

Comment:
You can just do `.where(userid: current_user.id)` instead of `.where('userid = ?', current_user.id)`

Reply:
Or even `where(user: current_user)`

Reply:
nice, that looks a lot nicer all ready!